cachelib/allocator/CacheAllocatorConfig.h

```diff
@@ -199,6 +201,25 @@ class CacheAllocatorConfig {
   // cachePersistence()
   CacheAllocatorConfig& usePosixForShm();
 
+  // Configures cache memory tiers. Accepts vector of MemoryTierCacheConfig.
+  // Each vector element describes configuration for a single memory cache tier.
+  // Tiers can be set up as ratios of total cache size or have their sizes
+  // explicitly specified. If ratios are used, then total cache size must be set
+  // before this method is called; if sizes are explicitly specified, then
+  // their sum sizes must match the total cache size if it's previously set; if
+  // the total size has not been set then this method will set it as a sum of
+  // all tier sizes
+  CacheAllocatorConfig& configureMemoryTiers(const MemoryTierConfigs& configs);
+
+  // Sets total cache size and configures cache memory tiers. This method
```

Comment:
does this override the previously set cache size ? And by cache size we mean the DRAM cache size ?

Reply:
By cache size we mean "total cache size" which is the sum of all memory tier sizes. By default there's only one DRAM memory tier whose size matches total cache size.

I decided to delete this 2 argument configureMemoryTiers method.